cli/ballerina-cli/src/main/resources/cli-help/ballerina-new.help

```diff
@@ -34,24 +33,24 @@ DESCRIPTION
        specified, this command creates a package with the default template.
 
 
-       OPTIONS
-            -t <template-name>, --template <template-name>
-                Create a package using a predefined template.
+OPTIONS
+    -t <template-name>, --template <template-name>
+        Create a package using a predefined template.
 
-                 There are three built-in templates named:
-                 1. `main`
-                 2. `service`
-                 3. `lib`
+        There are three built-in templates named:
+        1. `main`
+        2. `service`
+        3. `lib`
 
-       EXAMPLES
-            Create a package with the 'main' function.
-               $ bal new <package-name>
+EXAMPLES
+    Create a package with the 'main' function.
```

Comment:
Other commands have more space for indentation of the explanation.

Reply:
Fixed